_posts/theme/2019-12-23-Flora.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: theme
+title: Flora
+category: theme
```

Comment:
Hi, about https://github.com/typora/typora-theme-gallery/pull/96/files#r360929547

I mean 
1. Move _posts/theme/2019-12-23-Flora.md to _posts/fork/2019-12-23-Flora.md
1. Change `category: theme` to `category: fork`
2. Add your theme's link, screenshot and summary on _posts/theme/2019-03-04-Ursine.md